modules/installation-gcp-tested-machine-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// gcp modules
+
+[id="installation-gcp-tested-machine-types_{context}"]
+= Tested instance types for GCP
+
+The following Google Cloud Platform instance types have been tested with {product-title}.
+
+.Machine types
```

Comment:
1. Please replace "Machine types" with "Machine series", because the "N1", "N2", and etc. do be machine series and each has predefined machine types (see [Google docs](https://cloud.google.com/compute/docs/machine-types)).
2. Please replace the machine series "T2D" with "Tau T2D", which is the official name.

Reply:
@jianli-wei you can change the name by yourself, like this PR https://github.com/openshift/installer/pull/5517

Reply:
> 2\. Please replace the machine series "T2D" with "Tau T2D", which is the official name.

FYI the name issue is tracked by https://github.com/openshift/installer/pull/6296, thanks!

Reply:
> 1. Please replace "Machine types" with "Machine series", because the "N1", "N2", and etc. do be machine series and each has predefined machine types (see [Google docs](https://cloud.google.com/compute/docs/machine-types)).
> 
>     2. Please replace the machine series "T2D" with "Tau T2D", which is the official name.

I've replaced "Machine types" with "Machine series" for GCP. Thank you